kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseFileDataItem.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 201y Red Hat, Inc. and/or its affiliates.
```

Comment:
ah, ok :) I thought the `y` represents the last digit of current year, but it somehow wasn't replaced by the actual value.